Engines/Wine/Verbs/xact/script.js

```diff
@@ -11,50 +11,51 @@ include("engines.wine.plugins.regsvr32");
  * @returns {Wine} Wine object
  */
 Wine.prototype.xact = function () {
-    var that = this;
```

Comment:
I removed `var that = this;` in favor of arrow functions which allows us the direct usage of `this`